static/css/normalize.css

```diff
@@ -41,8 +41,8 @@ hgroup,
 main,
 menu,
 nav,
-section,
-summary {
+section
```

Comment:
normalize.css is something well pull as-is, so we shouldn't change it, otherwise changes will be lost on update.

Reply:
Where can I override that setting for \<summary\>?   The existing CSS breaks display of the turning triangle for collapsible sections....

Reply:
The Hugo-specific way of dealing with \<details\> tags seems to have eliminated the need to change normalize.css, so I've taking that commit out of the PR.